backend/server/routers/user.py

```diff
@@ -28,12 +29,25 @@ def set_user(token: str, item: Storage, overwrite: bool = False):
 @router.post("/saveLocalStorage/")
 def save_local_storage(localStorage: LocalStorage, token: str = DUMMY_TOKEN):
     # TODO: turn giving no token into an error
-    # THINK: this is wierd security wise, because we are setting it to what we are given no ma
+    planned: list[str] = sum((sum(year.values(), []) for year in localStorage.planner['years']), [])
```

Comment:
Can't vouch for what this line does as I don't understand it, but otherwise everything looks good to me.